patches/telemetry.diff

```diff
@@ -104,6 +104,13 @@ Index: code-server/lib/vscode/src/vs/workbench/services/telemetry/browser/teleme
 -				sendErrorTelemetry: this.sendErrorTelemetry,
 -			};
 -			this.impl = this._register(new BaseTelemetryService(config, configurationService, productService));
+-
```

Comment:
@code-asher I refreshed all the patches. Does this look okay to you?

Reply:
Looks like the blocks changed a little, probably due to using a different diff algorithm, but I think it should be fine!  Looks like I am using `patience` (you can see what you have by running `git config diff.algorithm` in the code-server repo).

Reply:
Uh...looks like I'm not using one? 😅 Reminds me of when I first started at Coder and Anmol/I were using different algorithms.

<img width="1322" alt="image" src="https://user-images.githubusercontent.com/3806031/187557241-104ed3c9-c997-494a-ba2b-5a959bdf3083.png">

Okay I will leave as is for now then and merge this!

Reply:
Ah that probably means it is using the default (`myers`).